dev/run-tpcds-test

```diff
@@ -53,6 +53,7 @@ echo "Using \`spark-submit\` from path: $SPARK_HOME" 1>&2
 exec "${SPARK_HOME}"/bin/spark-submit \
   --driver-memory 5g \
   --class org.apache.spark.sql.execution.benchmark.TPCDSQueryValidator  \
+  --conf spark.ui.enabled=false \
```

Comment:
Maybe this configuration also needs to be added here.

https://github.com/apache/auron/blob/a0d94d4066c8d7406695584550dc2f9b969acae0/spark-extension-shims-spark/src/test/scala/org.apache.auron/BaseAuronSQLSuite.scala#L22